README.md

```diff
@@ -5,10 +5,56 @@
 
  A wiki that contain all osu! rulesets
 
+## Domain
+
+- Dev build (Some function in dark mode like color invert will not work.) : https://dev.rulesets.info/
+- Official build : https://rulesets.info/
+- Directly to a ruleset page : https://[rulesetname].rulesets.info/
+
 ## Development Status
 
 A website structure is almost finish. Current adding more ruleset to website.
 
+## Developing Rūrusetto codebase
+
+*If you don't want to developing a codebase but you want to contribute a wiki you can skip to [contributing](#contributing) part.*
```

Comment:
```suggestion
+ If you are only interested in contributing to wiki content, instead of the underlying backend, please skip to [contributing](#contributing). 
```